service/gateway/router.go

```diff
@@ -95,5 +103,45 @@ func (g *Gateway) registerHandler(r *mux.Router) {
 		Name(replicateObjectPieceRouterName).
 		Methods(http.MethodPut).
 		HandlerFunc(g.replicatePieceHandler)
+
+	// path style
```

Comment:
The registerHandler function holds too many handlers. It is best to split it into different registerHandler functions.